apps/linkedin-to-notion/src/components/LinkedInNotionSidePanel/LinkedInNotionSidePanelContent.tsx

```diff
@@ -39,7 +39,8 @@ export const LinkedInNotionSidePanelContent = ({
   chrome.runtime.onMessage.addListener((msg) => {
     if (msg === 'updateLinkedInNotionSidePanel') {
       setLinkedInProfileInformation(null);
-      injectLinkedInInformation();
+      // We set a timeout to wait for the page to be fully loaded before attempting to scrape the information
+      setTimeout(() => injectLinkedInInformation(), 2000);
```

Comment:
For me the problem is that even though we send the message onComplete state of the tab, the message is fired to soon and the scraping is done with previous data. 

I honestly don't like the idea of the timeout and if we have to go for it I'd rather go for it in this file: `apps/linkedin-to-notion/src/background/index.ts`

Reply:
Yeah I really hate the idea too but I could find a better option. I've explored many (webNavigation, webRequest, window.onLoad, etc.) to no avail. I'm okay to move it in the service worker though! (and I'm more than open to a robust solution to the issue, that's for sure).

Reply:
cf. comment above + adding a timeout:
```suggestion
  if (tab.url && tab.url.match(linkedInURLRegex)) {
    if (status === 'complete') {
      return setTimeout(() => chrome.tabs.sendMessage(tabId, 'updateLinkedInNotionSidePanel'), 2000);
    }
  } else {
    return chrome.tabs.sendMessage(tabId, 'closeLinkedInNotionSidePanel');
  }
```